docs/code-culture.md

```diff
@@ -164,3 +164,66 @@ Example of good framework log messages (a little bit simplified) can look like e
   "message": "Task $taskId of Pod $podId was killed"
 }
 ```
+
+## Testing
+
+Tests are executable documentation and should be **written, read and maintained** as such. A test tells it's reader a short story about how some part of the system should behave given a certain input. Hundreeds of excelent books and articles are dedicated to writing proper tests so we'll try to avoid repeating them here.
```

Comment:
```suggestion
Tests are executable documentation and should be **written, read and maintained** as such. A test tells its reader a short story about how some part of the system should behave given a certain input. Hundreds of excellent books and articles are dedicated to writing proper tests, so we'll try to avoid repeating them here.
```

Reply:
it's vs its is the worst. Every time I read "it's" I mentally read aloud "it is".

Reply:
It's (!) my kryptonite 😞